app/models/post.rb

```diff
@@ -1,5 +1,13 @@
 class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments
   has_many :likes
-  has_many :Comments
+
+  def increment_user_posts_count
+    author.increment!(:posts_counter)
+  end
```

Comment:
[OPTIONAL] Nice work with this method :+1: It would be even better if you made use of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) to call this method, specifically the `after_save` callback. What it does is call the method whenever a new post is saved to the database automatically. 
To use it, you can use the following line: 
```
after_save increment_user_posts_count 
```
It's also best practice to make the method in question a private method.